### DISCUSSION TITLE
[Question]: A workaround for importing .jwpub files with partial manual import

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#3428] [Q&A] [answered]
### What is your question?

With one of the latest updates JW Library received the "Download" button when trying to open a video in a jwpub file, which is not already directly built-in into the file but rather must be downloaded (or previously manually imported).

### Is your question related to a problem?

When user imports a big jwpub event program, for example pioneer school media, most of the videos are being imported and that's great, but there's a set of videos that should be manually imported.

So it there any existing or potential workaround to enable fetching such videos from the Meeting Media Manager? For example, first fetch all the media, including the placeholder thumbnails (and make it visually clear that it required the download) and then download them with a one click?

Hopefully it's technically possible.

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing discussions to make sure my question is not a duplicate.

**Comment by mtdvlpr** [comment, marked as answer]
Duplicate of #3427
